opentasks-provider/src/main/java/org/dmfs/provider/tasks/handler/RelationHandler.java

```diff
@@ -248,13 +248,5 @@ else if (type == Relation.RELTYPE_CHILD)
                 db.update(TaskDatabaseHelper.Tables.TASKS, taskValues, Tasks._ID + "=" + oldValues.getLong(relIdCol), null);
             }
         }
-        else if (type == Relation.RELTYPE_SIBLING)
```

Comment:
It's ok to remove the dead branch, but the comment should be left for reference. It's still an unsolved issues.

Reply:
sure, will revert this change :wink: